src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -1077,11 +1077,11 @@ class RideHailAgent(
           stall.parkingZoneId
         )
     }
-    startRefueling(tick, triggerId, data, triggers)
+    startRefueling(tick, triggers)
   }
 
-  def startRefueling(tick: Int, triggerId: Long, data: RideHailAgentData, triggers: Seq[ScheduleTrigger]): Unit = {
-    handleStartRefuel(tick, triggerId, data, triggers)
+  private def startRefueling(tick: Int, triggers: Seq[ScheduleTrigger]): Unit = {
+    handleStartRefuel(tick, triggers)
```

Comment:
You removed `tick` in `handleStartRefuel`, but are passing it in. I think you meant to remove `tick` here also - and should keep `triggerId`

Reply:
Nice catch